### PR TITLE
init: Initial Gatsby Project Setup

### PR DESCRIPTION
**Pull request by CIPHERTron**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [X] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [X] Check the commit's or even all commits' message styles matches our requested structure.
- [X] Check your code additions will fail neither code linting checks nor unit test.
- [X] I have added necessary documentation (if appropriate)

### Description
- This PR sets up a basic gatsby project with Eslint and Prettier config.
- Basic Project Structure setup with components and pages directory

❤️Thank you!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gatsby-starter-hello-world",
```
Comment (abhibhaw) [inline package.json:2]: @CIPHERTron Don't forget to change name and description next time 😆

**Review by abhibhaw — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (CIPHERTron) [reply package.json:2]: @abhibhaw BIG OOPS XD

**Comment by CIPHERTron** [comment]
@all-contributors add @abhibhaw for Review

**Comment by CIPHERTron** [comment]
@all-contributors add @CIPHERTron for Code

**Pull request merged**